client-patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/Makefile.linux b/Makefile.linux
```

Comment:
Shouldn't it be part of repo with source code?

Reply:
Yes, but in a future release.  I'm creating this flatpak based on the stable release as advised.